plugins/modules/api_info.py

```diff
@@ -136,6 +136,7 @@
         - ip proxy
         - ip route
         - ip route vrf
+        - ip vrf
```

Comment:
This needs to be moved to the right position (lexicographically sorted), i.e. after `ip upnp interfaces`. Same in the other file.